setup/kubernetes/aws.md

```diff
@@ -36,6 +36,16 @@ EC2 instance from AWS'
 > [container-based virtual machines (CVMs)](../../workspaces/cvms.md) unless
 > you're running Coder in a bare-metal Kubernetes environment.
 
+### Single vs. multi-zone clusters
+
+In our experience, single-region, single-zone clusters work best for Coder.
```

Comment:
what evidence is this based on? various enterprise customers expect to deploy Coder in a multi-region configuration to increase Coder's resiliency. this sounds antonymous to that expectation. @mterhar thoughts?